### ISSUE TITLE
Data mismatch in Network monitoring page

### ISSUE THREAD

**Issue by sridharmeganathan** [LiskHQ/lisk-desktop#2872]
There is a data mismatch when i compare the lisk-desktop networking monitoring page with lisk explorer monitoring page.

- Versions distribution & Height distribution should be sorted from current to previous
- Versions distribution & Height distribution tooltip is not fully visible.
- Connected peers and disconnected peers can be changed.
- Connected peers grid should be sorted by height (descending)

![image](https://user-images.githubusercontent.com/51332105/80722726-faf6e700-8aff-11ea-893a-8c65e1042dbe.png)

![image](https://user-images.githubusercontent.com/51332105/80722816-15c95b80-8b00-11ea-88e5-4cf4c0326f1e.png)

![image](https://user-images.githubusercontent.com/51332105/80722872-2974c200-8b00-11ea-9c59-1ebd864a4101.png)

**Comment by reyraa** [comment]
I've fixed
- Versions distribution & Height distribution tooltip is not fully visible.
- Connected peers grid should be sorted by height (#2881 )

regarding the suggestion to sort Versions distribution legends by version number, I checked, it sorts by distribution. I think it makes more sense as it is, cause the graph is named Versions distribution, and it makes sense that everything in it is sorted by distribution. You want to know which versions the most nodes are running against not which version is the latest. or at least this is my take.
